CHANGELOG.md

```diff
@@ -1,3 +1,2 @@
-## 0.0.1
-
-* TODO: Describe initial release.
+## 0.0.2
+add message type cast to kotlin
```

Comment:
for the future - u should add new version and description instead of replacing old

Reply:
yeah, as it was in the first place. I just though Andrei ask me to remove the first line with old ver